### PR TITLE
Minor: updates specificity weights to reflect updated tutorial. Add cascade file

### PR DESCRIPTION
**Pull request by estelle**
Currently updating /en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance. This update reflects the change in that tutorial

### CODE REVIEW

**Comment by estelle** [comment]
https://github.com/mdn/content/pull/17118

File: learn/cascade/cascade-layers.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <style class="editable">
+@layer firstLayer, secondLayer;
+
+p { /* 0-0-1 */
+  background-color: red;
+  color: grey !important;
+  border: 5px inset purple;
+  margin: 15px;
+}
+p#addSpecifity { /* 1-0-1 */
```
Comment (dipikabh) [inline learn/cascade/cascade-layers.html:21]:
```suggestion
p#addSpecificity { /* 1-0-1 */
```

File: learn/cascade/cascade-layers.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  border-style: solid !important;
+}
+
+@layer firstLayer {
+  #addSpecifity { /* 1-0-0 */
```
Comment (dipikabh) [inline learn/cascade/cascade-layers.html:26]:
```suggestion
  #addSpecificity { /* 1-0-0 */
```

File: learn/cascade/cascade-layers.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    background-color: blue;
+    color: white !important;
+    border-width: 5px;
+    border-style: dashed !important;
+  }
+}
+
+@layer secondLayer {
+  p#addSpecifity { /* 1-0-1 */
```
Comment (dipikabh) [inline learn/cascade/cascade-layers.html:35]:
```suggestion
  p#addSpecificity { /* 1-0-1 */
```

File: learn/cascade/cascade-layers.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    border-style: dotted !important;
+  }
+}
+
+    </style>
+</head>
+
+<body>
+  <section class="preview">
+        <p id="addSpecifity">
```
Comment (dipikabh) [inline learn/cascade/cascade-layers.html:48]:
```suggestion
        <p id="addSpecificity">
```

File: learn/cascade/cascade-layers.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+    <textarea class="playable playable-css" style="height: 280px;">
+@layer firstLayer, secondLayer;
+
+p { /* 0-0-1 */
+  background-color: red;
+  color: grey !important;
+  border: 5px inset purple;
+}
+p#addSpecifity { /* 1-0-1 */
```
Comment (dipikabh) [inline learn/cascade/cascade-layers.html:61]:
```suggestion
p#addSpecificity { /* 1-0-1 */
```

File: learn/cascade/cascade-layers.html [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+  border-style: solid !important;
+}
+
+@layer firstLayer {
+  #addSpecifity { /* 1-0-0 */
```
Comment (dipikabh) [inline learn/cascade/cascade-layers.html:66]:
```suggestion
  #addSpecificity { /* 1-0-0 */
```

File: learn/cascade/cascade-layers.html [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+    background-color: green;
+    color: orange !important;
+    border-width: 10px;
+    border-style: dotted !important;
+  }
+}     
+    </textarea>
+
+    <textarea class="playable playable-html" style="height: 140px;">
+<p id="addSpecifity">
```
Comment (dipikabh) [inline learn/cascade/cascade-layers.html:85]:
```suggestion
<p id="addSpecificity">
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
just the 'specificity' typo, it is quite a tongue-twister as well 🙂

**Comment by teoli2003** [comment]
@estelle: can you resolve the conflict here? That way @dipikabh can re-review it, then merge it.

For the moment we have a 404 on MDN, and this leads to some confusion.

When this PR lands, it will fix mdn/content#17456

**Review by dipikabh — CHANGES_REQUESTED** [review]
Suggestion to number the examples (sorry, I just realized that this review had been sitting as 'Pending')

File: learn/cascade/specificity-boxes.html [hunk 8]
```diff
@@ -81,37 +81,37 @@
 
     <textarea class="playable playable-css" style="height: 280px;">
 
-/* specificity: 0101 */
+/* specificity: 1-0-1 */
```
Comment (dipikabh) [inline learn/cascade/specificity-boxes.html:84]:
```suggestion
/* 1. specificity: 1-0-1 */
```

File: learn/cascade/specificity-boxes.html [hunk 9]
```diff
@@ -81,37 +81,37 @@
...
 }
         
-/* specificity: 0201 */
+/* specificity: 2-0-1 */
 #outer #inner a {
     background-color: blue;
 }
 
-/* specificity: 0104 */
+/* specificity: 1-0-4 */
```
Comment (dipikabh) [inline learn/cascade/specificity-boxes.html:94]:
```suggestion
/* 3. specificity: 1-0-4 */
```

File: learn/cascade/specificity-boxes.html [hunk 10]
```diff
@@ -81,37 +81,37 @@
...
 #outer div ul li a {
     color: yellow;
 }
 
-/* specificity: 0113 */
+/* specificity: 1-1-3 */
```
Comment (dipikabh) [inline learn/cascade/specificity-boxes.html:99]:
```suggestion
/* 4. specificity: 1-1-3 */
```

File: learn/cascade/specificity-boxes.html [hunk 11]
```diff
@@ -81,37 +81,37 @@
...
 #outer div ul .nav a {
     color: white;
 }
 
-/* specificity: 0024 */
+/* specificity: 0-2-4 */
```
Comment (dipikabh) [inline learn/cascade/specificity-boxes.html:104]:
```suggestion
/* 5. specificity: 0-2-4 */
```

File: learn/cascade/specificity-boxes.html [hunk 12]
```diff
@@ -81,37 +81,37 @@
...
 div div li:nth-child(2) a:hover {
     border: 10px solid black;
 }
 
-/* specificity: 0023 */
+/* specificity: 0-2-3 */
```
Comment (dipikabh) [inline learn/cascade/specificity-boxes.html:109]:
```suggestion
/* 6. specificity: 0-2-3 */
```

File: learn/cascade/specificity-boxes.html [hunk 13]
```diff
@@ -81,37 +81,37 @@
...
 div li:nth-child(2) a:hover {
     border: 10px dashed black;
 }
 
-/* specificity: 0033 */
+/* specificity: 0-3-3 */
```
Comment (dipikabh) [inline learn/cascade/specificity-boxes.html:114]:
```suggestion
/* 7. specificity: 0-3-3 */
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Review by dipikabh — APPROVED** [review]
Thanks, @estelle!

**Pull request merged**